app/controllers/declaration/WarehouseIdentificationController.scala

```diff
@@ -74,6 +74,8 @@ class WarehouseIdentificationController @Inject()(
       case _                         => warehouseIdentificationPage(mode, form)
     }
 
-  private def updateCache(formData: WarehouseIdentification)(implicit request: JourneyRequest[AnyContent]): Future[ExportsDeclaration] =
-    updateDeclarationFromRequest(model => model.copy(locations = model.locations.copy(warehouseIdentification = Some(formData))))
+  private def updateCache(formData: WarehouseIdentification)(implicit request: JourneyRequest[_]): Future[ExportsDeclaration] =
+    updateDeclarationFromRequest { declaration =>
+      declaration.copy(locations = declaration.locations.copy(warehouseIdentification = Some(formData), inlandOrBorder = None))
```

Comment:
Where does this requirement come from to rest on the Warehouse page (I don't see it on the ticket's ACs) :thinking:

Reply:
I added this because when a value is entered on the Warehouse page, the /inland-or-border page is always skipped.
[The condition is here](https://github.com/hmrc/customs-declare-exports-frontend/blob/c8807cb680f5d036ee17b24bfae4c60e06706e1b/app/controllers/helpers/InlandOrBorderHelper.scala#L40)

Reply:
Ok cool. Can you add it as an AC on the ticket so that the QAs know to test it. Thanks :+1: